packages/table/src/locator.ts

```diff
@@ -51,14 +51,33 @@ export class Locator implements ILocator {
 
     private grid: Grid;
 
-    private rowHeaderWidth = 0;
-    private columnHeaderHeight = 0;
-
-    private numFrozenRows = 0;
-    private numFrozenColumns = 0;
-
-    public constructor(private tableElement: HTMLElement, private bodyElement: HTMLElement) {
-        // empty constructor
+    // these values affect how we map a mouse coordinate to a cell coordinate.
+    // for instance, a click at (0px,0px) in the grid could map to an arbitrary
+    // cell if table is scrolled, but it will always map to cell (0,0) if there
+    // are active frozen rows and columns.
+    private numFrozenRows: number;
+    private numFrozenColumns: number;
+
+    // the root table element within which a click is deemed valid and relevant.
+    private tableElement: HTMLElement;
+
+    // the scrollable element that wraps the cell container.
+    private scrollContainerElement: HTMLElement;
+
+    // the element containing all body cells in the grid (excluding headers).
+    private cellContainerElement: HTMLElement;
+
+    public constructor(
+        tableElement: HTMLElement,
+        scrollContainerElement: HTMLElement,
+        cellContainerElement: HTMLElement,
```

Comment:
you can declare `private` _and_ document these properties in signature...

```js
public constructor(
    /** Documentation */
    private property: Type;
) {
```

Reply:
Okey doke.